plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/CreateCommand.java

```diff
@@ -248,7 +248,10 @@ else if (heartConfig.createCenterSphere) {
 
             this.guildManager.spawnHeart(this.plugin.getGuildEntityHelper(), guild);
             //player.teleport(guildLocation);
-            guild.teleportHome(player);
+
```

Comment:
Please, delete this unnecessary empty line. This is not in line with needs of our bussiness right now. If it will be needed at any point in the future, I'll notify you so you can enter it again at later time. Thank you very much for your contribution, much appreciated. You're great!